### ISSUE TITLE
[Bug] New install unable to add ip address, clamav-daemon not running

### ISSUE THREAD

**Issue by tfinalx** [hestiacp/hestiacp#4151]
### Describe the bug

Install in fresh debian 11 and 12
1.  clamav-daemon show red icon on hestiacp server setting
2. Unable to add new ip. (Error: user hestiamail doesn't exist)

![2023-11-10 at 08-20-43](https://github.com/hestiacp/hestiacp/assets/5638725/dca87e08-b00a-4ca8-944b-31cdf54be49a)
![2023-11-10 at 08-21-12](https://github.com/hestiacp/hestiacp/assets/5638725/26e7ee89-60bb-484e-94b2-386b140565af)

### Tell us how to replicate the bug

1. Install a new one and look for server setting
2. Try to add a new ip

### Which components are affected by this bug?

Control Panel Installation or Upgrade

### Hestia Control Panel Version

1.8.10

### Operating system

Debian 11 & 12

**Comment by nazmulmarketer** [comment]
Yes, new version adding ip not working because not showing assign user list's

**Comment by nazmulmarketer** [comment]
How can i fix it?

**Comment by jaapmarcus** [comment]
> ### Describe the bug
> Install in fresh debian 11 and 12
> 
> 1. clamav-daemon show red icon on hestiacp server setting
> 2. Unable to add new ip. (Error: user hestiamail doesn't exist)
> 
> ![2023-11-10 at 08-20-43](https://private-user-images.githubusercontent.com/5638725/281916659-dca87e08-b00a-4ca8-944b-31cdf54be49a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0Njg3MjAsIm5iZiI6MTcwMTQ2ODQyMCwicGF0aCI6Ii81NjM4NzI1LzI4MTkxNjY1OS1kY2E4N2UwOC1iMDBhLTRjYTgtOTQ0Yi0zMWNkZjU0YmU0OWEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDFUMjIwNzAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjY5ZjQ1NGVmMDFhMTRkNDI4MGY3MWExMDU0NDcwMjM5NGY2MWU2OTQwMTNjNTgyNTIxZDZhYmY3NTQzMzBjNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.5vzeV8ZUZNtcZWjOMAj4thw_gdQTZVNFlJY_T9Uf0D0) ![2023-11-10 at 08-21-12](https://private-user-images.githubusercontent.com/5638725/281916665-26e7ee89-60bb-484e-94b2-386b140565af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0Njg3MjAsIm5iZiI6MTcwMTQ2ODQyMCwicGF0aCI6Ii81NjM4NzI1LzI4MTkxNjY2NS0yNmU3ZWU4OS02MGJiLTQ4NGUtOTRiMi0zODZiMTQwNTY1YWYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIwMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMDFUMjIwNzAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODdkNDVmZDAyODFmNmIwMGNiM2JmMTNhMmNhYzFmMTNhZDY5MjYyMDE0NzZlNGM4NzVkMmY1NWZhYWIwYjk0OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.xhvU5b7akCXLeDXXGHZDzNcWv1YXlLhshhAbFml3VpU)
> 
> ### Tell us how to replicate the bug
> 1. Install a new one and look for server setting
> 2. Try to add a new ip
> 
> ### Which components are affected by this bug?
> Control Panel Installation or Upgrade
> 
> ### Hestia Control Panel Version
> 1.8.10
> 
> ### Operating system
> Debian 11 & 12
> 
> ### Log capture
> _No response_

This bug should be fixed right now ...

For clamav check enough memory is availble

**Comment by jaapmarcus** [comment]
> How can i fix it?

That is a different bug ...

2023/12/01 22:08:43 [error] 144516#0: *206 FastCGI sent in stderr: "PHP message: PHP Warning:  Undefined array key "v_owner" in /usr/local/hestia/web/add/ip/index.php on line 51; PHP message: PHP Fatal error:  Uncaught TypeError: Hestiacp\quoteshellarg\quoteshellarg(): Argument #1 ($arg) must be of type string, null given, called in /usr/local/hestia/web/add/ip/index.php on line 51 and defined in /usr/local/hestia/web/inc/vendor/hestiacp/phpquoteshellarg/src/Hestiacp/quoteshellarg/quoteshellarg.php:14
Stack trace:
#0 /usr/local/hestia/web/add/ip/index.php(51): Hestiacp\quoteshellarg\quoteshellarg()
#1 {main}
  thrown in /usr/local/hestia/web/inc/vendor/hestiacp/phpquoteshellarg/src/Hestiacp/quoteshellarg/quoteshellarg.php on line 14" while reading response header from upstream, client: xxx, server: _, request: "POST /add/ip/ HTTP/2.0", upstream: "fastcgi://unix:/run/hestia-php.sock:", host: "xxx:8083", referrer: "https://xxx:8083/add/ip/"

**Comment by jaapmarcus** [comment]
Add It via CLI works fine

**Comment by arielesp** [comment]
1.8.11 fresh install is nothing possible add IP by panel

using CLI works fine

**Comment by sjpr-online** [comment]
For the clamav-daemon status:
- edit the file /usr/local/hestia/bin/v-list-sys-services
- change line 219 
`from: get_srv_state "$ANTIVIRUS_SYSTEM"`
`to: get_srv_state "$ANTIVIRUS_SYSTEM" "$proc_name"`

If you have postgres it will show the service down too. It is another error on the same file.
- Change line 272
`from: proc_name='postmaster'`
`to: proc_name='postgres'`

**Comment by tvoltolini88** [comment]
For the clamav-daemon status:

 - edit the file /usr/local/hestia/bin/v-list-sys-services
 - change line 219
`from: get_srv_state "$ANTIVIRUS_SYSTEM"`
`to: get_srv_state "$ANTIVIRUS_SYSTEM" "$proc_name"`

If you have postgres it will show the service down too. It is another error on the same file.

- Change line 272
`from: proc_name='postmaster'`
`to: proc_name='postgres'`

Edit: this works for Ubuntu 22, might work as well on Debian

**Comment by arielesp** [comment]
Hi, I edited the file /usr/local/hestia/web/add/ip/index.php
to

// List users
exec(HESTIA_CMD . "v-list-sys-users 'json'", $output, $return_var);

// fix?
$output = implode(", ", $output);
$output = rtrim($output, ',');
preg_match_all('/"([^"]+)"/', $output, $matches);
$users = $matches[1];
